hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/validation/ValidationCondition.java

```diff
@@ -47,7 +47,7 @@ public boolean shouldApply(OMRequest req, ValidationContext ctx) {
   OLDER_CLIENT_REQUESTS {
```

Comment:
We can rename OLDER_CLIENT_REQUEST to something OLDER_CLIENT_BUCKETLAYOUT_CHECK

Reply:
Done.